Quake/Makefile

```diff
@@ -70,7 +70,6 @@ define do_strip
 	$(call cmd_strip,$(1));
 endef
 endif
-CFLAGS += -std=gnu99
```

Comment:
On 9/13/16, Axel Gneiting <notifications@github.com> wrote:
>> -CFLAGS += -std=gnu99
>
> Are you sure this is fine? There are still plenty of functions where I
> declare variables not at the beginning of the scope?

Yes, however without an explicit -ansi or similar, gcc extensions
are in effect, so unless someone attempts using gcc2 (which btw
does not support -std=gnu99), you are fine.